TrafficCapture/trafficCaptureProxyServerTest/src/main/java/org/opensearch/migrations/trafficcapture/JMeterLoadTest.java

```diff
@@ -23,23 +26,49 @@
 
 public class JMeterLoadTest {
 
+    static class Parameters {
+        @Parameter(required = true,
+                names = {"-p", "--port"},
+                description = "Port number")
+        int backsidePort;
+        @Parameter(required = true,
+                names = {"-s", "--server-name"},
+                description = "Server name")
+        String domainName;
+    }
```

Comment:
http vs https will eventually be a concern too.  We could also take in a URL and parse those three components.  That's what the replayer or the proxy (IIRC) does, but it's weird because the path and query string wouldn't be allowed/used.

Reply:
On this note, are the tests run to test HTTPS or HTTP?  HTTPS should be the default.  I can't tell from the HttpSampler interface how to set it other than by specifying a fullUrl, which we aren't doing.

Reply:
The HttpSampler has a setProtocol method, which by default is set to HTTP, added a parameter for that in next commit.